src/lib/components/Dialog.react.js

```diff
@@ -17,6 +17,26 @@ type Props = {
   ariaLabelledBy?: string,
   /** Children to render inside of the Dialog */
   children?: Node,
+  /** The classes to be applied to this component. This keys in this object must be valid CSS rule
+   * names, and the values must be strings for the classnames to be assigned to each rule name
+   * Valid rule names are:
+   *   root
+   *   container
+   *   paper
+   *   scrollPaper
+   *   scrollBody
+   *   paperScrollPaper
+   *   paperScrollBody
+   *   paperWidthFalse
+   *   paperWidthXs
+   *   paperWidthSm
+   *   paperWidthMd
+   *   paperWidthLg
+   *   paperWidthXl
+   *   paperFullWidth
+   *   paperFullScreen
+   * */
+  classes?: Object,
```

Comment:
You can do this as type hinting to make smarter python checks:
```js
{
  root?: string,
  container?: string,
  paper?: string,
  ...
}
```

Reply:
Good idea. Thanks. All updated now.